utils/src/treev2/quickPickWizard/common/PickFilter.ts

```diff
@@ -0,0 +1,20 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import * as vscode from "vscode";
+
+export interface PickFilter<TPick = vscode.TreeItem> {
+    /**
+     * Filters for nodes that match the final target.
+     * @param node The node to apply the filter to
+     */
+    isFinalPick(node: TPick): boolean;
+
+    /**
+     * Filters for nodes that could have a descendant matching the final target.
```

Comment:
"have a descendant matching..." or "have an ancestor matching..."

Reply:
To match the function name I think it could be something like this:
"Filters for nodes that could be an ancestor of a node matching the final target."